src/main/resources/db/migration/V011__opprett_mellomlagring_konsistensavstemming.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE mellomlagring_konsistensavstemming
+(
+  id                    UUID PRIMARY KEY,
+  fagsystem             VARCHAR(4)                              NOT NULL,
```

Comment:
Trenger vi størrelse, spiller ingen rolle for postgres.

Reply:
Jeg satte størrelse som et ekstra skydd for at man ikke skal skrive inn feil data (ettersom fagsystem er maks 4 langt)